tutorials/plot_dipole_orientations.py

```diff
@@ -0,0 +1,197 @@
+
+The role of dipole orientations in distributed source localization
+==================================================================
+
+When performing source localization in a distributed manner (MNE/dSPM/sLORETA),
+the source space is defined as a grid of dipoles that spans a large portion of
+the cortex. These dipoles have both a position and an orientation. In this
+tutorial, we will look at the various options available to restrict the
+orientation of the dipoles and the impact on the resulting source estimate.
+"""
+
+###############################################################################
+# Loading data
+# ------------
+# Load everything we need to perform source localization on the sample dataset.
+
+from mayavi import mlab
+import mne
+from mne.datasets import sample
+from mne.minimum_norm import make_inverse_operator, apply_inverse
+
+data_path = sample.data_path()
+evokeds = mne.read_evokeds(data_path + '/MEG/sample/sample_audvis-ave.fif')
+left_auditory = evokeds[0].apply_baseline()
+fwd = mne.read_forward_solution(
+    data_path + '/MEG/sample/sample_audvis-meg-eeg-oct-6-fwd.fif',
+    surf_ori=True)
+noise_cov = mne.read_cov(data_path + '/MEG/sample/sample_audvis-cov.fif')
+subjects_dir = data_path + '/subjects'
+
+###############################################################################
+# The source space
+# ----------------
+# Let's start by examining the source space as constructed by the
+# :func:`mne.setup_source_space` function. Dipoles are placed along fixed
+# intervals on the cortex, determined by the ``spacing`` parameter. The source
+# space does not define the orientation for these dipoles.
+
+lh = fwd['src'][0]  # Visualize the left hemisphere
+verts = lh['rr']  # The vertices of the source space
+tris = lh['tris']  # Groups of three vertices that form triangles
+dip_pos = lh['rr'][lh['inuse'].astype(bool)]  # The position of the dipoles
+white = (1.0, 1.0, 1.0)  # RGB values for a white color
+gray = (0.5, 0.5, 0.5)  # RGB values for a gray color
+red = (1.0, 0.0, 0.0)  # RGB valued for a red color
+
+mlab.figure(size=(600, 400), bgcolor=white)
+
+# Plot the cortex
+mlab.triangular_mesh(verts[:, 0], verts[:, 1], verts[:, 2], tris, color=gray)
+
+# Mark the position of the dipoles with small red dots
+mlab.points3d(dip_pos[:, 0], dip_pos[:, 1], dip_pos[:, 2], color=red,
+              scale_factor=1E-3)
+
+mlab.view(azimuth=180, distance=0.25)
+
+###############################################################################
+# Fixed dipole orientations
+# -------------------------
+# While the source space defines the position of the dipoles, the inverse
+# operator defines the possible orientations of them. One of the options is to
+# assign a fixed orientation. Since the neural currents from which MEG and EEG
+# signals originate flows mostly perpendicular to the cortex [1]_, restricting
+# the orientation of the dipoles accordingly is a valid option that simplifies
+# the computation.
```

Comment:
It's potentially more than this. It can arguably help localization, too, at least in the case of accurate geometry + coreg as you're [constraining the problem in a better way](http://onlinelibrary.wiley.com/wol1/doi/10.1002/hbm.20155/full) (but probably not as good as loose)